.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: python
 
-python:
-    - "2.7"
+python: "2.7"
 
 sudo: false
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
+    directories:
+        - $HOME/.pip-cache/
+        - _build
```

Comment:
this caches the build documentation of previous builds with the same target branch. Sphinx detects changes from the last build and then executes a partial build. This will make the build process a lot quicker.